docs/experimental-module-policy.md

```diff
@@ -0,0 +1,35 @@
+# Drupal Core Experimental module policy
+
+Drupal 8 core introduced the concept of [experimental modules][1].
+These are modules that are provided with Drupal core for testing purposes, but that are not yet fully supported.
+They offer a wide range of functionalities, from migration management
+to site layout management.
+
+Due to the experimental nature of the modules from the OpenEuropa platform,
+we need to define a set of policies on their use inside OpenEuropa Components.
```

Comment:
This sentence moves from 'experimental nature' to the OpenEuropa platform, it seems to imply that the modules provided by OpenEuropa are also experimental. This could be reworded a bit to make it clearer.

Reply:
Agree, it took even me some time to understand what I meant.